docs/source/tutorial/tutorial-query-ui.md

```diff
@@ -300,31 +247,23 @@ query LaunchList {
 
 When you recompile, if you look in `API.swift`, you'll see a new nested type, `Mission`, with the two properties you requested.
 
-Because `missionPatch` returns a URL string to an image that can be displayed, you want to make it straightforward to display an image from its URL.
-
-To save time, go to **File > Swift Packages > Add Package Dependency...** and add a dependency on [SDWebImage](https://github.com/SDWebImage/SDWebImage). Paste in the URL to its git repo, `https://github.com/SDWebImage/SDWebImage.git`, and then select the main `SDWebImage` from the list of package products Xcode offers you once it resolves the repo.
-
-When SDWebImage finishes installing, go back to `MasterViewController.swift` and add the following import to the top of the file:
+Go back to `LaunchesViewController.swift` and add the following import of one of the libraries that was already in your project to the top of the file:
```

Comment:
This appears to already be in the start project, can this be removed?

Reply:
Good catch, [I removed it from the starter project](https://github.com/apollographql/iOSTutorial/commit/5624b5305696460de8aed2929be32f190e5d7aaa) so that this bit can stay (and make it clear you're using a lib and not UIKit)